integration-tests/src/common/registration.rs

```diff
@@ -27,6 +27,7 @@ pub fn do_registration(temp_dir: &TempDir) -> RegistrationResult {
         std::env::var("STAKE_SKEY").unwrap_or_else(|_| "STAKE_SKEY not defined".to_owned());
     let stake_vkey =
         std::env::var("STAKE_VKEY").unwrap_or_else(|_| "STAKE_VKEY not defined".to_owned());
+    let vote_skey = std::env::var("VOTE_SKEY").ok();
```

Comment:
IMHO for tests it's better to generates voting key rather than take it from env

Reply:
My intention was to allow specifying it but not require it. If the env variable is not set it will pass a `None` value to the command which will generate one on its own

Reply:
Oh ok, didn't notice that. sorry.